### ISSUE TITLE
[VoiceOver] Hide images/icons in settings from VoiceOver

### ISSUE THREAD

**Issue by aDENTinTIME** [AudioBooth/AudioBooth#298]
In the settings menu the images/icons should be hidden from VoiceOver and instead appropriate alt text should be read.
